dev.sh

```
@@ -8,295 +8,217 @@ export CARGO_INCREMENTAL=1
 export CARGO_NET_RETRY=10
 export CARGO_NET_TIMEOUT=60
 
+# Configuration
+DEMO_TIMEOUT=${DEMO_TIMEOUT:-60s}
+
+# Helper functions
+check_clean() {
+    if [[ "$@" == *"--clean"* ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)